app/src/test/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentSheetControllerTest.kt

```diff
@@ -593,7 +599,15 @@ class ScheduleAppointmentSheetControllerTest {
         schedulers = TrampolineSchedulersProvider()
     )
 
-    whenever(protocolRepository.protocol(protocolUuid)).thenReturn(protocol)
+    if(protocol != null) {
```

Comment:
Shouldn't we avoid branching in tests?

Reply:
Can we create two separate methods instead? Also, we should break this method if possible.

Reply:
Yes, I can do this.

Reply:
Done.